packages/mandelbrot/views/partials/navigation/information.nunj

```diff
@@ -5,11 +5,21 @@
                 {{ frctl.theme.get('labels.info') }}
             </div>
         </div>
-        <div class="Tree-aside">
-            {{ frctl.theme.get('labels.builtOn') }}:
-            <time datetime="{{ frctl.theme.get('now').toISOString() }}">
-                {{ frctl.theme.get('now').toLocaleDateString(frctl.theme.get('lang')) }}
-            </time>
-        </div>
+        <dl class="Tree-aside">
```

Comment:
Shall we add a condition around the wrapper to not display an empty one if no information is provided?

Reply:
i thought about this, but then it would make more sense for the user to remove information from the navigation panels.

Reply:
Then again, dumb-proofing it so that it wouldn't error out makes sense, too...